src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -86,16 +86,19 @@ limitations under the License.
 
     <attribute side="server" code="0x0002" define="SUBJECTS_PER_ACCESS_CONTROL_ENTRY" type="int16u" writable="false">
       <description>SubjectsPerAccessControlEntry</description>
+        <quality persistence= "fixed" />
```

Comment:
could we fix alignment of these to match same-level tags? This seems more indented than the `description` above.